templates/scripts/restart-authservices.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+ENMASSE_NAMESPACE=${1:-enmasse-infra}
+MINREADY=${2:-0}
+MINAVAILABLE=$(($MINREADY + 1))
+
+function wait_deployment_ready() {
+    local dep=$1
+    local minReady=$2
+
+    ready=0
+    while [[ "${ready}" -lt "${minReady}" ]]
+    do
+        ready=`kubectl get deployment ${dep} -o jsonpath='{.status.readyReplicas}' -n ${ENMASSE_NAMESPACE}`
+        if [[ "${ready}" -lt "${minReady}" ]]; then
+            sleep 5
```

Comment:
I'd report the actual number ready on each iteration, otherwise the operator may just end up beliving a script is hung.